src/client/helper/parser.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
```

Comment:
When adding files, make sure to add them to CMakeLists.txt, otherwise CMake won't compile them and the build will fail. Also consider having all client-specific files in one directory, as long as it's not too many we won't need a helper directory.
Does the code currently build for you? With clang I get undefined symbol errors, apparently it does not generate symbols for the Parser class in parser.cpp since it does not realize that class is needed in other files. It works after including parser.h in it (and changing the code so that the class is not defined twice). Might just be a clang-specific thing though.

Sorry if this is a lot

Reply:
I still can't get windows to recognize that asio is installed so I've been coding blind. I added a CMakeLists.txt file within the helper class so it should work now.